File: October 2024/October 1/grading_students.py
def grading_students(grades):
    final_grades = []

    for i in range(len(grades)):
        if grades[i] < 38:
            final_grades.append(grades[i])
        else:
            if (grades[i]+1)%5 == 0:
                grades[i]+=1
            elif (grades[i]+2)%5 == 0:
                grades[i]+=2
            final_grades.append(grades[i])

    return final_grades

print(grading_students([73, 67, 38, 33]))

File: October 2024/October 1/gradingStudents.js
function gradingStudents(grades) {
    let finalGrades = []
  
    for(let i = 0; i < grades.length; i++){
      if(grades[i] < 38){
        finalGrades.push(grades[i])
      }
      else{
        if((grades[i] + 1)%5 == 0){
          grades[i]+=1
        }else if((grades[i] + 2)%5 == 0){
          grades[i]+=2
        }
        finalGrades.push(grades[i])
      }
    }
    
    return finalGrades
  }
  console.log(gradingStudents([73, 67, 38, 33]))  